repository_service_tuf_api/targets.py

```diff
@@ -172,3 +172,26 @@ def delete(payload: DeletePayload) -> Response:
     return Response(
         data=data, message="Remove Target(s) successfully submitted."
     )
+
+
+def post_publish_targets() -> Response:
+    task_id = get_task_id()
+    repository_metadata.apply_async(
+        kwargs={
+            "action": "publish_targets",
+            "payload": None,
+        },
+        task_id=task_id,
+        queue="rstuf_internals",
```

Comment:
That's the name of the message queue name where the task should go, right?

Reply:
Yes, we have [two queues in RSTUF.](https://repository-service-tuf.readthedocs.io/projects/rstuf-worker/en/latest/devel/index.html#adding-removing-targets)
The publish targets are used in the `rstuf_internals`

Reply:
Cool that we have documentation about this.